optuna/_gp/prior.py

```diff
@@ -20,13 +20,16 @@ def default_log_prior(kernel_params: "gp.KernelParamsTensor") -> "torch.Tensor":
     # Log of prior distribution of kernel parameters.
 
     def gamma_log_prior(x: "torch.Tensor", concentration: float, rate: float) -> "torch.Tensor":
-        # We omit the constant factor `rate ** concentration / factorial(concentration)`.
+        # We omit the constant factor `rate ** concentration / Gamma(concentration)`.
         return (concentration - 1) * torch.log(x) - rate * x
 
     # NOTE(contramundum53): The parameters below were picked qualitatively.
     # TODO(contramundum53): Check whether these priors are appropriate.
     return (
```

Comment:
```suggestion
    # NOTE(contramundum53): Prior for inverse_squared_lengthscales was determined by heuristic.
    return (
```

Reply:
Well, every prior here are determined by heuristic, and that's no special about `inverse_squared_lengthscales`. I thought `The parameters below were picked qualitatively.` was talking about this.

Reply:
I changed the sentence to `The priors below were picked by heuristics.`